docs/admin/install.rst

```diff
@@ -830,29 +840,29 @@ After enabling it, you should set :setting:`ENABLE_HTTPS` in the settings:
 Set properly SECURE_HSTS_SECONDS
 ++++++++++++++++++++++++++++++++
 
-If your site is served over SSL, you have to consider setting a value for :setting:`django:SECURE_HSTS_SECONDS`
+If your site is served over SSL, consider setting a value for :setting:`django:SECURE_HSTS_SECONDS`
 in the :file:`settings.py` to enable HTTP Strict Transport Security.
-By default it's set to 0 as shown below.
+By default it is set to 0 (as shown below).
 
 .. code-block:: python
 
    SECURE_HSTS_SECONDS = 0
 
 If set to a non-zero integer value, the :class:`django:django.middleware.security.SecurityMiddleware`
-sets the :ref:`django:http-strict-transport-security` header on all responses that do not already have it.
+applies the :ref:`django:http-strict-transport-security` header to all responses without it.
 
 .. warning::
 
-    Setting this incorrectly can irreversibly (for some time) break your site. Read the
```

Comment:
How is it irreversible, but only for some time?
Needs to be undone via the terminal?